noncompiled/JsonSerializer.js

```diff
@@ -68,30 +80,41 @@ JsonDeserializer.plate = function (plate_json, _world, options) {
 
 	return plate;
 }
-JsonDeserializer.world = function (world_json, options) {
+JsonDeserializer.world = function (world_json, grid, options) {
 	options = options || {};
 
 	var _world = new World(
 	{
-		grid: view.grid,
+		grid: grid,
```

Comment:
step in the right direction, could probably use world.grid to simplify method signature.

Reply:
I was thinking about sending in the current global "world" (or maybe the global "model") in as an argument to JsonDeserializer.model() and JsonDeserializer.world() instead of "grid", but I didn't see anything else that I needed to copy from the pre-load datastructure other than the grid.